README.md

```diff
@@ -23,6 +23,21 @@ model behaviors and take the mystery out of model development. Kolena helps you:
 This `kolena` package contains the Python client library for programmatic interaction with the Kolena ML testing
 platform.
 
+## Setup
```

Comment:
This looks like instruction for development. Should we include the instruction for users wanting to install kolena through `pip`?

Reply:
Example 7 [here](https://pip.pypa.io/en/stable/cli/pip_install/#examples)